src/docsfy/generator.py

```
@@ -288,8 +288,12 @@ async def run_planner(
         ai_cli_timeout=ai_cli_timeout,
     )
 
-    plan = parse_json_response(output)
+    plan = parse_json_response(_strip_ai_artifacts(output))
     if plan is None:
+        logger.warning(
+            f"[{project_name}] Planner output is not valid JSON. "
+            f"Raw output ({len(output)} chars): {output[:500]}"
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)